react/src/lib/DatePicker/DatePickerDay/index.js

```diff
@@ -45,11 +45,13 @@ class DatePickerDay extends React.Component {
           `${(isOutsideMonth && ` md-datepicker__day--outside-month`) || ''}`
         }
         onClick={this.handleClick}
-        ariaLabel={`${getDate(day)}`}
+        ariaLabel={`${day.format("dddd, MMMM Do YYYY")}`}
         aria-selected={isSelected}
         tabIndex={-1}
       >
-        {getDate(day)}
+        <div aria-label={day.format("dddd, MMMM Do YYYY")}>
```

Comment:
Believe this still needs to be ariaLabel?

Reply:
I initially thought so as well but a flag was thrown up when compiling. Found this in the [React documentation](https://reactjs.org/docs/accessibility.html):

> Note that all aria-* HTML attributes are fully supported in JSX. Whereas most DOM properties and attributes in React are camelCased, these attributes should be hyphen-cased (also known as kebab-case, lisp-case, etc) as they are in plain HTML